scripts/reporting/issues.py

```diff
@@ -108,7 +109,13 @@ def __init__(self, verbose, issue, metadata):
         first_field = self.body["requestError"].split('","')[1]
         field_row = metadata[metadata["field_name"] == first_field]
         self.form = field_row["form_name"].item()
-        instrument = self.body["experiment_site_id"].split("-")[0]
+        experiment_site_id_regex = (
+            f"({utils.STUDY_ID_REGEX}_{utils.EVENT_REGEX}_)?(.*)-"
+        )
+        instrument = re.match(
```

Comment:
this will throw an error if the match is not found.

Reply:
Ok I will check if the match call is empty before getting the instrument.